src/liftdrag_plugin/liftdrag_plugin.cpp

```diff
@@ -311,19 +306,22 @@ void LiftDragPlugin::OnUpdate()
 
   // compute cl at cp, check for stall
   double cl;
+  // make simulation more realistic by assuming some losses due to the finite nature of
+  // the wing. The value is chosen arbitrarily.
+  const double finite_plate_cl_factor = 0.7;
```

Comment:
Can you make this a parameter?

Reply:
I would advocate against it. If the 0.7 won't work for a simulation, I don't think any value will make the aerodynamics model work for that simulation.

Also, it would be hard to explain to users to what value they should set this. The real factor won't be constant. Anyone who is able to determine a better factor is probably not happy with the rest of the simplifications in the aerodynamic model. That is why I think a fixed value is fine here.